tests/settings.py

```diff
@@ -1,12 +1,24 @@
 import os
 import sys
 
+from distutils.version import StrictVersion
+import django
+django_version_string = django.get_version()
+if django_version_string.startswith('2.0.dev'):
+    DJANGO_VERSION = StrictVersion('2.0')
```

Comment:
This is gross, but I needed a way to not run oauth tests if not supported by the current django version.  I'm open to other ideas.